lambdas/db-migration/package.json

```diff
@@ -19,7 +19,7 @@
     "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles"
   },
   "dependencies": {
-    "@cumulus/db": "2.0.1-alpha.1",
+    "@cumulus/db": "3.0.1-alpha.1",
```

Comment:
Where is this used?

Reply:
Hmm, a fine question.   I updated this in response to my lerna issue, but assumed it was supposed to be there.   Looking.

Reply:
It doesn't seem to be used that I can see

Reply:
Actually, yes it is. My bad.